Buildings/Applications/DHC/CentralPlants/Cooling/Controls/ChilledWaterPumpSpeed.mo

```diff
@@ -0,0 +1,91 @@
+within Buildings.Applications.DHC.CentralPlants.Cooling.Controls;
+model ChilledWaterPumpSpeed
+  "Controller for variable speed chilled water pumps"
```

Comment:
Headed pump / dedicated pump? Also, clarify that it is only applicable for maximum 2 pumps.

Reply:
Could you please clarify "headed pump" vs. "dedicated pump"?

Reply:
Please see below the difference between headed and dedicate pumps: 
- Headed pump:
<img width="338" alt="image" src="https://user-images.githubusercontent.com/23491921/94751157-95bc5180-033c-11eb-9e5d-4d069444a1b4.png">

- Dedicated pump:
<img width="281" alt="image" src="https://user-images.githubusercontent.com/23491921/94751164-9e148c80-033c-11eb-862d-eb88b6327224.png">

Reply:
Thanks! It's designed for headed pumps.